autometa/taxonomy/ncbi.py

```diff
@@ -333,22 +311,18 @@ def parse_nodes(self):
         return nodes
 
     def parse_merged(self):
-        """Short summary.
+        """
+        Parses through `merged.dmp` databse and returns a dictionary having old and new taxid
 
         Parameters
         ----------
-        verbose : boolean
+        self.verbose : boolean
             prints progress to terminal (the default is False).
 
         Returns
         -------
         dict
-            {child_taxid:{'parent':parent_taxid,'rank':rank}, ...}
-
-        Raises
-        -------
-        ExceptionName
-            Why the exception is raised.
+            returns a merged dictionary in the form {old_taxid: new_taxid}
```

Comment:
The dictionary is not _merged_ with anything else and this may be misleading. This just parses the `merged.dmp` file. Second, the return type is already specified in line 324, so saying this returns a dictionary is redundant.

```suggestion
            {old_taxid: new_taxid, ...}
```